src/core/connection.h

```diff
@@ -254,6 +254,8 @@ typedef struct QUIC_CONN_STATS {
 
         uint64_t TotalBytes;            // Sum of UDP payloads
         uint64_t TotalStreamBytes;      // Sum of stream payloads
+
+        uint64_t AcksReceived;          // Count of receive ACK frames.
```

Comment:
Don't put Received in the name. Maybe, call it `ValidAckFrames` and put it after `ValidPackets`.